modules/puppet_connect/plans/test_input_data.pp

```diff
@@ -13,6 +13,7 @@
 #
 plan puppet_connect::test_input_data(TargetSpec $targets = 'all') {
   $targs = get_targets($targets)
+  $plugin_hook = $targs[0].plugin_hooks
```

Comment:
This will fail if there are no targets. Should we guard against that?

Reply:
Or maybe we should check this against the whole set of targets rather than comparing each individual target to the first target?

```puppet
$targs.map |$targ| { $targ.plugin_hooks }.unique.length > 1
```